WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -426,6 +427,19 @@ class ReaderDiscoverViewModel @Inject constructor(
         }
     }
 
+    fun updateScreenVisibility(isScreenVisible: Boolean) {
+        this.isScreenVisible = isScreenVisible
+    }
+
+    private fun trackContentPresentedOnce() {
+        if (!isContentPresentedTrackedOnce) {
+            isContentPresentedTrackedOnce = true
```

Comment:
❓ Do we intentionally update the `isContentPresentedTrackedOnce` even when the screen is not visible?

Reply:
Yes, I am doing that intentionally. What I try to achieve is that when the user enters the screen and then leaves right after, the event gets triggered and the content arrives, even if they are not looking, so I count it as track and miss, since I don't want to count it on any other occurrence afterwards.

Does this makes sense or I am mumbling nonsense here? 😊

Reply:
Ahh, I see. In that case, I'd consider renaming the variable. It feels a bit confusing that `isContentPresentedTrackedOnce == true` but the event was never actually tracked. Wdyt?

Perhaps something like "isFirstTimeContentPresented"?

Reply:
Yeap, I was thinking about it as well, I like your suggestion, will do! 🌟 

PS: The way I was thinking about it was since this variable is used within the `trackContentPresentedOnce()` function, which means that the caller wanted to track the content presented, then I count it as such, as once, whether that actually did the tracking based on the screen being visible is an implementation detail down the line for the caller code to not worry about it. I agree it is tricky...

Reply:
PS: I am not going to do any changes until we decide to move forward with this tracking on Android. If we do, I'll make the change, else I'll just close the PR and delete the branch.